students/humberto_gonzalez/session09/donor_models.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Jul 14 19:23:00 2019
+
+@author: Humberto Gonzalez
+"""
+
+import os
+import pickle
+import tempfile
+
+class Donor:
+    def __init__(self,name):
+        """
+        donor is initialized with a name and empty donations list
+        """
+        self.name = name
+        self.donations = []
+    
+    def add_donation(self,donation):
+        """
+        adds donation to the donations list
+        """
+        try:
+            donation = float(donation)
+            self.donations.append(donation)
+        except:
```

Comment:
make sure to catch specific exception - I'm assuming should be around float() call?